conbench/app/_plots.py

```diff
@@ -236,7 +263,15 @@ def _inspect_for_multisample(items) -> tuple[bool, Optional[int]]:
     """
 
     # Get number of samples for each benchmark.
-    samplecounts = [len(i["data"]) for i in items]
+    try:
+        samplecounts = [len(i["data"]) for i in items]
+    except KeyError:
```

Comment:
I did this 'guard' because I have no confidence if `items` _really_ always is what I think it is. Type annotations and type checking will help with that.

Reply:
TODO note added